src/main/resources/script_templates/ImageJ2/Manual_Registration.py

```diff
@@ -0,0 +1,76 @@
+# @ImageJ ij
+# @Dataset ds
+# @OpService ops
+# @DatasetService datasetService
+# @OUTPUT Dataset final_dataset
+
+import math
+
+from net.imglib2.util import Intervals
+from net.imagej.axis import Axes
+from net.imglib2.view import Views
+
+from ij.plugin.frame import RoiManager
+
+# Initialize some variables
+img = ds.getImgPlus()
+rois = RoiManager.getInstance().getRoisAsArray()
+
+images = []
+
+total_x_shift = 0
+total_y_shift = 0
+
+# Iterate over regions of interest
+for j, r in enumerate(range(len(rois)-1)):
```

Comment:
Would it be more pythonic to directly use the ROIs instead of reusing the index, e.g. `for j, (start, end) in enumerate(izip(rois, rois[1:]))`? That would also replace the `start` and `end` assignment in the next lines.